### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Adebayo-George**
Here, a desktop version of the portfolio template was replicated. This time it was made to be responsive using the CSS media querry.

### CODE REVIEW

**Review by awais-amjed — COMMENTED** [review]
Hi @Adebayo-George , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 

## Invalid Code Review Request

You have submitted a project, that has linters failing.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+            <li>Ruby</li>
+          </ul>
+          <button class="my-project-button" type="submit">See Project</button>
+        </div>
+      </div>
+    </div>
+    <div class="all-cards">
+    <div id="first-card">
+      <div id="first-card-content">
+        <h2>Profesional Art Printing Data</h2>
```
Comment (RolandM99) [inline index.html:88]: - [ ] Kindly, add a new card in your work section on this empty space I mentioned bellow.
image

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,562 @@
...
+    width: 218%;
+  }
+
+  #about-myself {
+    position: relative;
+    top: 310px;
+    color: #fff;
+    display: flex;
+    justify-content: space-around;
+  }
```
Comment (RolandM99) [inline styles.css:535]: - [ ] Kindly, make sure that your about section for the desktop version is similar to the original design from Figma. take a look on the table below:

|Your Design|Figma Design|
|-------------|---------------|
|image|image|

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi @Adebayo-George, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

- [ ] [**REQUIRED**] I'll suggest you to follow every details(margins, font, padding,...) from the original design so that your project looks like the Figma Design and while implementing the desktop version, keep the mobile version intact so that your project still responsive. Make sure that you fit the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) before asking for a re-review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,563 @@
...
```
Comment (Wes-Isaac) [inline styles.css:530]: - [ ] As the previous reviewer recommended kindly fix the spacing.


### Figma's design
image

### Your design
Screen Shot 2022-09-02 at 5 11 03 PM

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,563 @@
...
+    background-size: 330px 501px;
+  }
+
+  .list {
+    list-style: none;
+    margin-left: -66px;
+    position: relative;
+    left: -367px;
+    top: 145px;
+  }
```
Comment (Wes-Isaac) [inline styles.css:466]: - [ ] [_OPTIONAL_] Please check the spacing for the icons on the design.

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,563 @@
...
+/* .section-header {
+  height: 216px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+} */
```
Comment (Wes-Isaac) [inline styles.css:335]: - [ ] [_OPTIONAL_] Kindly remove commented code, to keep your code clean.

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @Adebayo-George, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights

- Good commit message ✔️
- No Linter errors ✔️
- Follow Github flow ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Check the comments under the review._


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @Adebayo-George              :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for making the requested changes good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

- [ ] Kindly delete all commented out and unwanted/ unused pieces of code from your code base.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**